### PR TITLE
forms

### PR DESCRIPTION
**Pull request by maksym-yerokhin**
[DEMO LINK](https://maksym-erokhin.github.io/layout_html-form/)
[TEST REPORT LINK](https://maksym-erokhin.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,5 +13,165 @@
...
+        <legend>Personal information</legend>
+    
+        <label class="form-field">
+            Surname:
+          <input 
+             type="text" 
+             name="Surname" 
+             required
+             autocomplete="off"
+            >
```
Comment (solaryasha) [inline src/index.html:27]:
```suggestion
          <input 
             type="text" 
             name="Surname" 
             required
             autocomplete="off"
          >
```

File: src/index.html [hunk 2]
```diff
@@ -13,5 +13,165 @@
...
+    
+      <fieldset class="groups">
+        <legend>Additional info:</legend>
+    
+        <label class="form-field">
+           Comments:
+          <textarea name="feedbach" 
+            required
+            autocomplete="off"
+           >
```
Comment (solaryasha) [inline src/index.html:161]:
```suggestion
          <textarea 
            name="feedbach" 
            required
            autocomplete="off"
          >
```

File: src/index.html [hunk 3]
```diff
@@ -13,5 +13,165 @@
...
+          <input 
+             type="time" 
+             name="time_to_bed" 
+             required
+            >
+        </label>
+    
+        <label class="form-field">
+            What are you favorite brand of cars?
+          <select name="Cars" multiple required>
```
Comment (solaryasha) [inline src/index.html:133]:
```suggestion
          <select 
            name="Cars" 
            multiple 
            required
          >
```

**Review by solaryasha — CHANGES_REQUESTED** [review]
Test report demo link is not working
You should run test locally (`npm run test`)
Then deploy all together

**Review by solaryasha — CHANGES_REQUESTED** [review]
Все ще немає тестів

File: src/index.html [hunk 4]
```diff
@@ -13,5 +13,168 @@
   <body>
     <h1>HTML Form</h1>
```
Comment (solaryasha) [inline src/index.html:14]: You should remove this and you will pass all tests

File: src/index.html [hunk 5]
```diff
@@ -13,5 +13,168 @@
...
+            Do you love cats?
+          <input type="radio" 
+             name="cats" 
+             value="y" 
+             required> Yes 
+          <input type="radio" 
+             name="cats" 
+             value="n" 
+             required
+            > No
```
Comment (solaryasha) [inline src/index.html:111]: Check point 3 in [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

File: src/style.css [hunk 6]
```diff
@@ -1 +1,8 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin: 10px 0;
+}
+
+.section {
+  margin: 20px 0;
```
Comment (solaryasha) [inline src/style.css:7]: It's not enough to pass test, read 8 point in the [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

**Review by solaryasha — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 7]
```diff
@@ -11,7 +11,185 @@
...
+          >
+        </label>
+    
+        <label class="form-field">
+            What time do you go to bed?
+          <input 
+             type="time" 
+             name="time_to_bed" 
+             required
+             class="label-input"
```
Comment (solaryasha) [inline src/index.html:139]: Do you use somewhere this selector ?

**Review by solaryasha — APPROVED** [review]

File: src/index.html (hunk 7 above)
Comment (maksym-erokhin) [reply src/index.html:139]: > 

Oh no, I'll delete it now

**Pull request closed without merging**